libraries/SITL/SITL.cpp

```diff
@@ -475,9 +475,28 @@ const AP_Param::GroupInfo SIM::AirspeedParm::var_info[] = {
     // @Values: 0:Disabled, 1:Enabled
     // @User: Advanced
     AP_GROUPINFO("FAIL",    3, SIM::AirspeedParm,  fail, 0),
+    // @Param: ARSPD_FAILP
```

Comment:
The prefix is in the wrong spot, This should just be FAILP. The one level up where these are included you add the prefix. Like gripper for example. 

https://github.com/ArduPilot/ardupilot/blob/9b0639557338792356f50e39046e216d215be32b/libraries/SITL/SITL.cpp#L217-L219

Then the documentation should work for both airspeed 1 and 2.